.github/workflows/dashboard.yaml

```diff
@@ -0,0 +1,39 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+      - dashboard-setup
```

Comment:
I think this was just for testing the current PR, but we don't want it for the actual prod version?

Reply:
Yeah extactly.... Just for testing purpose